This patch removes a call to AM_ICONV() from the Autoconfigure sources for
Poppler. Removing this call prevents having to install GNU gettext just to
provide a M4 macro.

diff --git a/configure.ac b/configure.ac
index eb7403f..6525203 100644
--- a/configure.ac
+++ b/configure.ac
@@ -532,12 +532,6 @@ AC_ARG_ENABLE(poppler-cpp,
                              [Don't compile poppler cpp wrapper.]),
               enable_poppler_cpp=$enableval,
               enable_poppler_cpp="yes")
-if test x$enable_poppler_cpp = xyes; then
-  AM_ICONV()
-  if test x$am_func_iconv != xyes; then
-    enable_poppler_cpp=no
-  fi
-fi
 
 AM_CONDITIONAL(BUILD_POPPLER_CPP, test "x$enable_poppler_cpp" = "xyes")
 
